src/Compute/Compute/help/New-AzVmssConfig.md

```diff
@@ -109,6 +111,66 @@ Creates a VMSS configuration object. (autogenerated)
 New-AzVmssConfig -Location <String> -SkuCapacity 2 -SkuName 'Standard_A0' -UpgradePolicyMode Automatic -IdentityType SystemAssigned;
 ```
 
+### Example 3:
```

Comment:
@pselugar . Seems like the PS team has enabled some new checks on the examples in our .md files. They likely triggered for you because you are editing those files and the StaticAnalysis occurred on those edited .md files. 
You will need to resolve these issues in the example files shown in the errors onyour PR: https://dev.azure.com/azclitools/5147fa83-336e-44ef-bbe0-c86b8ae86cbb/_apis/build/builds/13367/logs/36